### PR TITLE
NEPT-690: Add maintenance offline template page.

### PR DESCRIPTION
**Pull request by Fefaine**

### CODE REVIEW

**Comment by drupol** [comment]
Commit syntax is wrong... use GrumPHP please :)

File: includes/alter/css.inc [hunk 1]
```diff
@@ -9,12 +9,12 @@
...
 function ec_europa_css_alter(&$css) {
-  $path_fancybox = libraries_get_path('fancybox');
-
-  unset(
-    $css[drupal_get_path('module', 'date') . '/date_api/date.css'],
-    $css[$path_fancybox . '/helpers/jquery.fancybox-buttons.css'],
-    $css[$path_fancybox . '/helpers/jquery.fancybox-thumbs.css'],
-    $css[$path_fancybox . '/jquery.fancybox.css']
-  );
+  if (function_exists('libraries_get_path') && ($path_fancybox = libraries_get_path('fancybox'))) {
```
Comment (drupol) [inline includes/alter/css.inc]: Open question: Should we use `function_exists()` or `module_exists()` ?

**Comment by drupol** [comment]
What about having 2 maintainance pages.

1 when we set the site into maintainance mode explicitely.
1 when the db is offline.

ex:

```
if (isset($variables['db_is_active']) && !$variables['db_is_active']) {
  $variables['theme_hook_suggestion'] = 'maintenance_page__offline';
} else {
  $variables['theme_hook_suggestion'] = 'maintenance_page';
}
```

**Comment by drupol** [comment]
Sorry forget what I said.

#MondayMorning

File: includes/alter/css.inc (hunk 1 above)
Comment (voidtek) [inline includes/alter/css.inc]: It's true that it's more beautiful.
It's tested and committed.

**Comment by Medi4tor** [comment]
+1

File: templates/page/maintenance-page--offline.tpl.php [hunk 2]
```diff
@@ -92,152 +45,157 @@
...
         <h1 class=" ecl-heading ecl-heading--h1 ecl-u-color-white">Site offline</h1>
       </div>
+    </div>
+  </div>
+  <div class="page-content">
+    <div class="ecl-container">
+      <a id="main-content" tabindex="-1"></a>
+      <div class="row">
+        <section class="section <?php print $content_column_class; ?>">
+          <h2><?php print t('This website is currently under maintenance'); ?></h2>
```
Comment (gillesdeudon) [inline templates/page/maintenance-page--offline.tpl.php]: As we are in a offline situation, the system is not available. We should use this translation function instead: get_t()?

Same remark for line 48

**Comment by drupol** [comment]
Needs to proper rebase.

**Pull request merged**